src/styles/lists.styl

```diff
@@ -107,6 +107,7 @@ sundown = #ffb3b3
             color sundown
             border rem(2) solid sundown
             background-color chablis
+        &.konnector-error--update::before,
         &.konnector-error--connection::before
```

Comment:
Those classes are used to put a red badge on the top right corner. I think it should be more obvious that this class is doing this. In the HTML, seeing something like `konnector-error--with-badge` would be clearer to me.